src/niswitch/examples/niswitch_connect_channels_grpc.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
+
+import argparse
+import grpc
+import niswitch
+import sys
+
+
+def example(resource_name, channel1, channel2, topology, simulate):
+    # if we are simulating resource name must be blank
+    resource_name = '' if simulate else resource_name
+    # Connect to an already running server using the following server information
+    server_address = "localhost"
+    server_port = "31763"
+    channel = grpc.insecure_channel(f"{server_address}:{server_port}")
+    session_options = niswitch.GrpcSessionOptions(channel, "", niswitch.SessionInitializationBehavior.AUTO)
```

Comment:
Preferably pull out the session name to a separate variable, maybe with a comment about empty string:
```python
session_name = ''  # user-specified name; empty string means use a new, unnamed session
```

Reply:
Makes sense to be more explicit for example. done